packages/token/src/token.js

```diff
@@ -141,6 +141,48 @@ class Token {
     return txHash
   }
 
+  /**
+   * Checks if a transaction is confirmed by checking if a number of blocks has
+   * been mined since the transaction was added.
+   * @param {string} txHash: the transaction hash.
+   * @param {number} numBlocks: the number of block confirmation to wait for
+   * @returns {Promise<{status:string, receipt:Object}>}
```

Comment:
nit: update to reflect it returns undefined in case of an error
```
@returns {Promise<{status:string, receipt:Object} || undefined>}
```

Reply:
👍